irods/manager/user_manager.py

```diff
@@ -46,10 +46,10 @@ def remove_quota(self, user_name, resource="total"):
         )
 
     def get(self, user_name, user_zone=""):
-        query = self.sess.query(User).filter(User.name == user_name)
+        if 0 == len(user_zone):
```

Comment:
do we prefer this style?

```suggestion
        if not user_zone:
```